### DISCUSSION TITLE
Alerts stop populating

### DISCUSSION THREAD

**Discussion by kennysden** [Security-Onion-Solutions/securityonion#2819] [Unsupported Versions]
Hi, 
 
Does anybody know why my elastalerts stop generating? 

Logstash logs
![image](https://user-images.githubusercontent.com/59517722/106516977-da502900-64a5-11eb-9fcf-3a5684488449.png)

**Comment by cm-ops** [comment]
@kbhill 

Looks like it is trying to write to a closed index. What does `curl -sk https://localhost:9200/_cat/indices` show?
